packages/next/build/swc/src/transform.rs

```diff
@@ -117,7 +117,27 @@ where
 {
     let c = get_compiler(&cx);
 
-    let src = cx.get::<JsString>(0)?.into_utf8()?.as_str()?.to_owned();
+    let unknown_src = cx.get::<JsUnknown>(0)?;
+    let src = match unknown_src.get_type()? {
+        napi::ValueType::String => napi::Result::Ok(
+            unsafe { unknown_src.cast::<JsString>() }
```

Comment:
Same here. We don't need unsafe as 

```rust
JsString::try_from(unknown_src)?
```

will work.

Reply:
yep, I changed it